clients/drcachesim/tools/reuse_distance.h

```diff
@@ -369,20 +372,21 @@ struct line_ref_list_t {
         else
             --dist; // Don't count self.
 
-        if (DEBUG_VERBOSE(0) && verify_skip_) {
-            // Compute reuse distance with a full list walk as a sanity check.
-            // This is a debug-only option, so we guard with DEBUG_VERBOSE(0).
-            // Yes, the option check branch shows noticeable overhead without it.
-            int_least64_t brute_dist = 0;
-            for (prev = head_; prev != ref; prev = prev->next)
-                ++brute_dist;
-            if (brute_dist != dist) {
-                std::cerr << "Mismatch!  Brute=" << std::dec << brute_dist
-                          << " vs skip=" << dist << "\n";
-                print_list();
-                assert(false);
-            }
-        }
+        IF_DEBUG_VERBOSE(
```

Comment:
Hmm, so this is the same as #ifdef DEBUG?

Reply:
Yes, but with the option of making it depend on knob_verbose...